test/unit/DCAHub/dca-hub-swap-handler.spec.ts

```diff
@@ -1012,8 +1027,8 @@ contract('DCAHubSwapHandler', () => {
               ratioAToB: BigNumber.from(ratioAToB),
               ratioBToA: BigNumber.from(ratioBToA),
               intervalsInSwap: SwapInterval.intervalsToByte(...intervalsInSwap),
-              totalAmountToSwapTokenA: constants.ZERO,
-              totalAmountToSwapTokenB: constants.ZERO,
+              totalAmountToSwapTokenA: BigNumber.from(ratioBToA),
```

Comment:
Why are you using these?

Reply:
I thought it would be easier to use these values instead of having to pass new parameters. We just need something that changes, so that we can test that it is exposed correctly